CHANGELOG.md

```diff
@@ -18,6 +20,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - tests: new Task targets `test:unit` (replaces target `test`) and `test:integration` (replaces target `test-integration`).
 - build: cleaner Taskfile.
 
+### Fixed
+
+- Return error text code in case of http code 500. 
```

Comment:
A "code" is a number, so we don't really return it in the text :-)
Maybe something like "Return standard error description in addition to the error code"

Reply:
Thanks for that, I'm doing the change.